packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -76,6 +76,10 @@ const propTypes = {
    * Callback function triggered when key is released.
    */
   onKeyUp: PropTypes.func,
+  /**
+   * Callback function triggered when mouse is pressed.
+   */
+  onMouseDown: PropTypes.func,
```

Comment:
From the Teams conversation, it sounds like using onMouseDown in this component is discouraged from an accessibility standpoint and may be removed in the future for enhance accessibility. In that case, we should avoid display this new prop in the props table either by:

1. Making this as a private prop.
2. Completely remove it as a first-class prop but would need to pass it to `handleMouseDown` for the callback.

Reply:
marked prop as private [0019324](https://github.com/cerner/terra-core/pull/3642/commits/00193249dfa8ae7cf535f4a41ae5239d52702fef)